docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1021 @@
+# encoding: utf-8
+require "logstash/{plugintype}s/base"
+require "logstash/namespace"
+
+# Add any asciidoc formatted documentation here
+# This example filter will replace the contents of the default
+# message field with whatever you specify in the configuration.
+#
+# It is only intended to be used as an example.
+class LogStash::{pluginclass}::{pluginnamecap} < LogStash::{pluginclass}::Base
+
+  # Setting the config_name here is required. This is how you
+  # configure this {plugintype} from your Logstash config.
+  #
+  # {plugintype} {
+  #   {pluginname} { message => "My message..." }
+  # }
+  config_name "example"
+
+  # Replace the message with this value.
+  config :message, :validate => :string, :default => "Hello World!"
+
+
+  public
+  def register
+    # Add instance variables
+  end # def register
+
+  public
+  def filter(event)
+
+    if @message
+      # Replace the event message with our message as configured in the
+      # config file.
+      event["message"] = @message
+    end
+
+    # filter_matched should go in the last line of our successful code
+    filter_matched(event)
+  end # def {plugintype}
+
+end # class LogStash::{pluginclass}::{pluginnamecap}
+----------------------------------
+endif::filter_method[]
+
+// /////////////////////////////////////////////////////////////////////////////
+// Output (conditionally recognized by the presence of the receive_method
+// attribute)
+// /////////////////////////////////////////////////////////////////////////////
+ifdef::receive_method[]
+[source,ruby]
+[subs="attributes"]
+----------------------------------
+# encoding: utf-8
+require "logstash/{plugintype}s/base"
+require "logstash/namespace"
+
+# Add any asciidoc formatted documentation here
+# An example output that does nothing.
+class LogStash::{pluginclass}::{pluginnamecap} < LogStash::{pluginclass}::Base
+  config_name "example"
+
+  public
+  def register
+  end # def register
+
+  public
+  def receive(event)
+  end # def receive
+
+end # class LogStash::{pluginclass}::{pluginnamecap}
+----------------------------------
+endif::receive_method[]
+
+=== Anatomy of a {plugintype} plugin
+
+Now let's take a line-by-line look at this new example plugin.
+
+==== The `encoding`
+
+It seems like a small thing, but please don’t omit adding
+
+[source,sh]
+----------------------------------
+# encoding: utf-8
+----------------------------------
+
+to the top of your {plugintype} plugin. Logstash depends on things being in
+UTF-8, so we put this here to tell the Ruby interpreter that we’re going to be
+using the UTF-8 encoding.
+
+==== `require` Statements
+
+A Logstash {plugintype} plugin requires some parent classes that can be
+referenced through the indicated statements. The following require statements
+are mandatory:
+
+[source,ruby]
+[subs="attributes"]
+----------------------------------
+require "logstash/{plugintype}s/base"
+require "logstash/namespace"
+----------------------------------
```

Comment:
Hmm.  This inline note hasn't been "cleared" in spite of the fact that I have re-rendered the text accordingly.